app/commands/git/sync_concept_exercise.rb

```diff
@@ -79,12 +79,12 @@ def find_concepts(slugs)
 
     memoize
     def author_usernames_config
-      head_git_exercise.authors.to_a.map { |a| a["exercism_username"] }
+      head_git_exercise.authors.to_a.map { |a| a[:exercism_username] }
```

Comment:
Rather than doing this, can we make authors a hash with indifferent access (you may have done this later - I'm just going through the PR commit-by-commit).